### ISSUE TITLE
Ability to stop and skip current timer

### ISSUE THREAD

**Issue by roman-yagodin** [ympavlov/minidoro#8]
It would be cool to have ability to stop (cancel) current timer (⏹️) and skip to the next one (⏭️).

UPD: I've noticed the ⏹️ button for work timer, but there are no such button for rest one!

**Comment by ympavlov** [comment]
Thank you for reply and sorry for big delay.

Minidoro follows the Pomodoro Technique very rigorously

According to the technique any activity except planned work is treated as a break. So you aren't able to break a "break" and you shouldn't start working before a break ends.
Although you can stop work timer if "you give in to temptation or something really urgent comes up" and start again later.

I appreciate your experience wtih Minidoro. But most probably, won't fix untill the concept changes
